udata/core/search/commands.py

```diff
@@ -32,7 +32,12 @@ def reindex(doc_type=None):
         if not doc_type or doc_type.lower() == doctype.lower():
             log.info('Reindexing {0} objects'.format(model.__name__))
             if es.indices.exists_type(index=name, doc_type=doctype):
-                es.indices.delete_mapping(index=name, doc_type=doctype)
+                log.warn(("The index {name} already exists and his mapping "
+                          "will not be updated with that command, you need "
+                          "to delete it manually (e.g. "
+                          "`curl -XDELETE 'http://localhost:9200/_all/'`) and "
+                          "to use the ` search init` command if you want to "
+                          "update the mapping.").format(name=name))
```

Comment:
I think you can only suggest to use `search init` which properly handle index deletion.